get-started/index.md

```diff
@@ -56,127 +56,156 @@ redirect_from:
 
 {% include_relative nav.html selected="1" %}
 
-Welcome! We are excited you want to learn how to use Docker.
+Welcome! We are excited you want to learn how to use Docker. Docker lets you
+easily **build, ship, and run** any application on any platform. Let's get
+started.
 
-In this six-part tutorial, you will:
+The Docker _Get Started_ tutorial teaches you how to:
 
-1. Get set up and oriented, on this page.
-2. [Build and run your first app](part2.md)
-3. [Turn your app into a scaling service](part3.md)
-4. [Span your service across multiple machines](part4.md)
-5. [Add a visitor counter that persists data](part5.md)
-6. [Deploy your swarm to production](part6.md)
+1. Set up your Docker environment (on this page)
+2. [Build an image and run it as one container](part2.md)
+3. [Scale your app to run multiple containers](part3.md)
+4. [Distribute your app across a cluster](part4.md)
+5. [Stack services by adding a backend database](part5.md)
+6. [Deploy your app to production](part6.md)
 
-The application itself is very simple so that you are not too distracted by
-what the code is doing. After all, the value of Docker is in how it can build,
-ship, and run applications; it's totally agnostic as to what your application
-actually does.
-
-## Prerequisites
-
-While we'll define concepts along the way, it is good for you to understand
-[what Docker is](https://www.docker.com/what-docker) before we begin.
-
-We also need to assume you are familiar with a few concepts before we continue:
-
-- IP Addresses and Ports
-- Virtual Machines
-- Editing configuration files
-- Basic familiarity with the ideas of code dependencies and building
-- Machine resource usage terms, like CPU percentages, RAM use in bytes, etc.
-
-Finally, though we'll remind you again when you need these things, you can
-save yourself some distraction at that time by [signing up for a
-Docker ID](https://cloud.docker.com) and using it on your local machine
-by running the following command:
-
-```
-docker login
-```
-
-## A brief explanation of containers
+## Images and containers
 
 An **image** is a lightweight, stand-alone, executable package that includes
-everything needed to run a piece of software, including the code, a runtime,
-libraries, environment variables, and config files.
+everything needed to run a piece of software&#8212;the code, a runtime,
```

Comment:
Let's try to avoid using weird HTML entities when we can...

Reply:
Wilco